tests/unit-tests/api-plugins-controller.php

```diff
@@ -85,159 +82,6 @@ public function test_get_plugins_authorized() {
 		$this->assertEquals( $expected_jetpack_info, $data['jetpack'] );
 	}
 
-	/**
```

Comment:
afaik, all these tests are not making any requests. `server->dispatch` is emulating the server receiving a request

Reply:
It's the way to test Rest API endpoints without needing http requests

Reply:
It's not about testing the REST API endpoints. The endpoint _handler_ is making requests to the plugin repository to fetch plugin code.

Reply:
gotcha